go.mod

```diff
@@ -7,7 +7,7 @@ replace github.com/graymeta/stow => github.com/kastenhq/stow v0.1.2-kasten
 require (
 	github.com/briandowns/spinner v1.21.0
 	github.com/golang/mock v1.6.0
-	github.com/kanisterio/kanister v0.0.0-20221021092747-4dca8c82ae9d
+	github.com/kanisterio/kanister v0.0.0-20230210192213-5c2302420309
```

Comment:
@pavannd1 why doesn't this use v0.89.0 ?

Reply:
We have not used released versions consistently but just commits off of master. This was a coincidence